session-5/session-5-part-2.ipynb

```diff
@@ -61,10 +61,12 @@
    ]
   },
   {
-   "cell_type": "markdown",
+   "cell_type": "code",
```

Comment:
This is the first code block that is about importing stuff. It should be a code cell instead of a markdown.